### ISSUE TITLE
Feature request: useCookie react hook

### ISSUE THREAD

**Issue by smashercosmo** [reactivestack/cookies#207]
Seems like time has come :)

**Comment by ElForastero** [comment]
Or just export CookieContext so we can use `useContext` hook.

```js
import React, { useContext } from 'react';
import { Context } from 'react-cookie';

const cookie = useContext(Context);
```

**Comment by revskill10** [comment]
Any update on this ?

**Comment by denichodev** [comment]
I've made a rough PR, might want to help check it https://github.com/reactivestack/cookies/pull/209

**Comment by eXon** [comment]
An official useCookies hook will be added to the library very soon. I'll keep this issue up-to-date.

**Comment by eXon** [comment]
It has been released in `v3.1`! See the doc.
